vignettes/lab_grading.Rmd

```diff
@@ -0,0 +1,395 @@
+---
+title: "Lab grading"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Lab grading}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+# Introduction
+
+Within the ADLB VAD there is a concept of lab grading, where there is a set of criteria 
+for particular lab tests that grade the severity or abnormality of a lab value. The 
+grades are from 1 to 4, and grade 0 can be viewed generally as a “NORMAL” value. The
+higher the grade the more severe or more abnormal the lab value is. There are several 
+sets of lab grading criteria, for the initial implementation of lab grading we will
+look at NCI-CTCAEv4. (In future releases we will look to implement further grading criteria,
+for example NCI-CTCAEv5)
+
+The NCI-CTCAE version 4 and 5 grading criteria can be found 
+here: https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm .
+
+The NCI-CTCAEv4 criteria can be found under the heading
+**Common Terminology Criteria for Adverse Events (CTCAE) v4.0**
```

Comment:
Could you hyperlink this to the section.  I think this is it https://ctep.cancer.gov/protocoldevelopment/electronic_applications/ctc.htm#ctc_40

Reply:
Done